idb/postgres/internal/writer/writer.go

```diff
@@ -141,13 +141,20 @@ func setSpecialAccounts(addresses transactions.SpecialAddresses, batch *pgx.Batc
 // (0 if not an asset or app transaction).
 func transactionAssetID(block *bookkeeping.Block, intra uint64, typeenum idb.TxnTypeEnum) uint64 {
 	assetid := uint64(0)
+	stxnad := block.Payset[intra]
 	txn := block.Payset[intra].Txn
```

Comment:
could you delete this since you are adding `stxnad`?

Reply:
txn usage here looks ok to me